pkgs/by-name/ca/cascadia-code/package.nix

```
@@ -2,6 +2,7 @@
   lib,
   stdenvNoCC,
   fetchzip,
+  useVariableFont ? true,
```

Defaulting to `true` will significantly change the default behavior of the package. IMHO it would be better to default to `false` for the time being.

```suggestion
  useVariableFont ? false,
```